slashing-protection/src/integration-test/java/tech/pegasys/web3signer/slashingprotection/WatermarkUpdatesIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
 import org.apache.tuweni.units.bigints.UInt64;
 import org.junit.jupiter.api.Test;
 
-public class WatermarkUpdatesIntegrationTest extends InterchangeBaseIntegrationTest {
+public class WatermarkUpdatesIntegrationTest extends IntegrationTestBase {
```

Comment:
that IntegrationTestBase renames could be a standalone PR, seems to have touched quite a bit (easy to review here, but touched many files)

Reply:
done